src/app/core/default-pattern-renderer/default-pattern-renderer.component.html

```diff
@@ -18,12 +19,12 @@
       <div *ngIf="directedPatternRelations?.length > 0">
         <ng-container *ngFor="let relation of directedPatternRelations">
           <p class="horiz-centered">
-            <a [routerLink]="['..',relation.sourcePatternId]">{{ relation?.sourcePatternName}}</a>
+            <a [routerLink]="['..',relation.sourcePatternUri]">{{ relation?.sourcePatternName}}</a>
```

Comment:
is this change intended?

Reply:
yes it was the reason for the error when routing via patternlinks and accessing the pattern via the info button in the graph.
Instead of accessing the pattern via: patternpedia/PL/Patternuri patternpedia/PL/P_ID got called - which does not exist